app/views/site/render_gene_expression_plots.js.erb

```diff
@@ -1,3 +1,17 @@
+<%
+=begin
+%>
```

Comment:
StackOverflow suggests 
```
<%#
   comment stuff
   more comment
%>
```
as a more concise and IDE-friendly way of doing block comments in erb

Reply:
Much better!  Thanks, done in a2f157e.